### PR TITLE
Docblock updates

### PR DESCRIPTION
**Pull request by ThomasLandauer**

### CODE REVIEW

File: src/Codeception/Lib/Interfaces/Web.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
      * $I->amOnPage('/register');
      * ```
      *
-     * @param $page
+     * @param string $page
```
Comment (DavertMik) [inline src/Codeception/Lib/Interfaces/Web.php]: For Yii2 module this can also be an array...

File: src/Codeception/Module/Doctrine2.php [hunk 2]
```diff
@@ -461,7 +461,7 @@ public function grabEntitiesFromRepository($entity, $params = [])
      * @version 1.1
      * @param $entity
      * @param array $params
-     * @return array
+     * @return object
```
Comment (DavertMik) [inline src/Codeception/Module/Doctrine2.php]: thanks!

File: src/Codeception/Lib/Interfaces/Web.php (hunk 1 above)
Comment (DavertMik) [inline src/Codeception/Lib/Interfaces/Web.php]: I think Yii2 module should override this method as well as the docblock (and documentation) to change this behavior.

**Comment by ThomasLandauer** [comment]
OK, so what's the bottom line? Should I change anything?  
I just wanted to get rid of `param null` which doesn't make any sense at all.

**Comment by DavertMik** [comment]
I think I should not forget to update the Yii2 module, that's all. Thanks for the update!

**Pull request merged**